Dockerfile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Using specific tag for now, there was some reason newer minio versions didn't work
-FROM quay.io/cloudservices/minio:RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff as minio-examples
+# Using specific tag to avoid newer minio versions that don't currently work
+FROM quay.io/minio/minio:RELEASE.2021-06-17T00-10-46Z as minio-examples
```

Comment:
I would feel a tad bit more comfortable if we used the exact same image -- with the `hotfix`, but pull it from `docker.io` instead of `quay.io/cloudservices`. 

We should not run into any rate limiting issues (this time), since we are not pulling that image in production, or even for testing, just once for building our own custom image.

```Bash
docker pull docker.io/minio/minio:RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff
...

docker pull quay.io/minio/minio:RELEASE.2021-06-17T00-10-46Z
...

docker images | grep minio

minio/minio          RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff   1377bf686561   2 years ago    351MB
quay.io/minio/minio  RELEASE.2021-06-17T00-10-46Z                    2c01061ad11a   2 years ago    339MB

```

Reply:
@rafvasq -- I went ahead with this change to get the build/test going